### ISSUE TITLE
Coveralls not trying to authorize with Github

### ISSUE THREAD

**Issue by jmagnusson** [lemurheavy/coveralls-public#550]
I've turned on `Third-party application access policy` in my organisation's settings and after this Coveralls doesn't try to re-authorize with GitHub. This is probably why I'm also experiencing the problem described in #539.

**Comment by jmagnusson** [comment]
What you're probably doing wrong is that you're not looking for the organization's access rights. When I revoked access for Coveralls on my personal account @ https://github.com/settings/applications I was finally redirected to Github's authorization page when I clicked on Coveralls' Repos tab. There I had the ability to grant Coveralls access to my organization. After that my org's repos showed up on the Repos page.

PS. It's kind of sad that you don't reply your paying customers. DS.

**Comment by nickmerwin** [comment]
hi @jmagnusson thanks for the report -- I've added a "re-authorize with github" button to our add repos page.

Thanks!

**Comment by tomekwi** [comment]
I struggled with the “Re-authorize with github” button but I managed to find a solution.

I decided to restrict access for third-party apps to some of my organizations. After that coveralls could no longer access my account even though I explicitly granted access to these orgs for the app “Coveralls”.

It turns out it’s not “Coveralls” (above) that needed the access granted, but “Coveralls Pro” (the one with a sneaky, dark logo below):

![image](https://cloud.githubusercontent.com/assets/4624660/9763408/a5392604-5709-11e5-8279-fc55c363d70c.png)

I lost well over an hour searching for the solution – simple but very unintuitive.

Hope it helps someone out there.

**Comment by dotnetCarpenter** [comment]
@nickmerwin The **"re-authorize with github" button** just redirects to my repo page https://coveralls.io/.
Is something broken? I have disabled **Third-party application access policy** - if I re-enable only travis appear. I have clicked the *Refresh Private Repos* and *Re-Authorize Github* multiple times, the first just shows a spinner for ever (30 minutes so far) and the latter just redirects me.

I'm trying to add svgdotjs/svg,js - any ideas?
